### PR TITLE
23310 - Facade

### PR DESCRIPTION
**Pull request by simonbu11**

### CODE REVIEW

File: eapim/policy.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            <openid-config url="{{oauth-eapim-config-url}}" />
+            <required-claims>
+                <claim name="aud">
+                    <value>{{oauth-spi-aud-claim}}</value>
+                </claim>
+            </required-claims>
+        </validate-jwt>
+
+        <set-header name="X-Functions-Key" exists-action="skip">
+            <value>{{squasher-function-app-key}}</value>
```
Comment (mmiddleton3301) [inline eapim/policy.xml]: You'll want to update this to be `eventbroker-function-app-key`.

**Review by mmiddleton3301 — CHANGES_REQUESTED** [review]
A very important thing to look at!

File: eapim/policy.xml (hunk 1 above)
Comment (simonbu11) [reply eapim/policy.xml]: Darn; copy/paste failed me again!

**Review by mmiddleton3301 — APPROVED** [review]
👀

**Pull request merged**